src/main/java/com/mcmoddev/basemetals/init/Recipes.java

```diff
@@ -65,14 +65,7 @@ protected static void initVanillaRecipes() {
 
 		// Iron items
 		if (Options.materialEnabled(MaterialNames.IRON)) {
-			material = Materials.getMaterialByName(MaterialNames.IRON);
-			oreDictName = material.getCapitalizedName();
-
-		// Not needed for 1.11.1+
-		/*
-			GameRegistry.addRecipe(new ShapelessOreRecipe(new ItemStack(material.getItem(Names.NUGGET), 9), Oredicts.INGOT + oreDictName));
-			GameRegistry.addRecipe(new ShapedOreRecipe(new ItemStack(material.getItem(Names.INGOT)), "xxx", "xxx", "xxx", 'x', Oredicts.NUGGET + oreDictName));
-		*/
+			// this should all be handled elsewhere
```

Comment:
This one shouldn't be changed *yet*, It's useful to keep for the purpose of backporting